backend/src/controllers/projects/index.ts

```diff
@@ -11,6 +11,18 @@ export const getAllProjects = async (req: Request, res: Response) => {
   }
 };
 
+export const getProject = async (req: Request, res: Response) => {
+  const { project_id } = req.params;
+  try {
+    const project = await db.select('*').from('projects').where({ id: project_id }).first();
+    project
+      ? res.status(StatusCodes.OK).json(project)
+      : res.status(StatusCodes.NOT_FOUND).json({ error: 'Project not found' });
```

Comment:
I would suggest you write it as such for readability:
```suggestion
    if(!project){
      res.status(StatusCodes.NOT_FOUND).json({ error: 'Project not found' });
      return;
    }
    res.status(StatusCodes.OK).json(project)
      : 
```